NAMESPACE

```diff
@@ -52,6 +52,8 @@ importFrom(stats,model.frame)
 importFrom(stats,terms)
 importFrom(stats,vcov)
 importFrom(stringr,boundary)
+importFrom(utils,capture.output)
```

Comment:
can we please use `utils::capture.output` etc. i.e. only import methods when needed, for normal functions just use `::`